src/force/CabanaPD_ForceModels_PMB.hpp

```diff
@@ -251,6 +251,137 @@ auto createForceModel( PMB, Fracture, ParticleType particles,
         delta, K, G0, temp, alpha, temp0 );
 }
 
+template <typename TemperatureType>
+struct ForceModel<PMB, Elastic, DynamicTemperature, TemperatureType>
+    : public ForceModel<PMB, Elastic, TemperatureDependent, TemperatureType>
+{
+    using base_type =
+        ForceModel<PMB, Elastic, TemperatureDependent, TemperatureType>;
+    using base_model = PMB;
+    using fracture_type = Elastic;
+    using thermal_type = DynamicTemperature;
+
+    using base_type::c;
+    using base_type::delta;
+    using base_type::K;
+
+    // Thermal parameters
+    using base_type::alpha;
+    using base_type::temp0;
+    double thermal_coeff;
+    double kappa;
+    double cp;
+
+    // Explicitly use the temperature-dependent stretch.
```

Comment:
What does it mean "explicitly use"?

Reply:
Because of the way this was written the temperature-independent version is always available - the temperature-dependent version must be explicitly used

Reply:
I see. I am wondering if eventually, having both options written in a consistent manner would be better. We may need something like that for bi-material systems. No need to change this at this time.

Reply:
When we support multiple materials of course you'll be able to change model parameters for each independently. I don't see what's inconsistent